compiler/testData/integration/smoke/noReflect/noReflect.kt

```diff
@@ -0,0 +1,5 @@
+package noReflect
+
+fun main() {
+    String::class.annotations
```

Comment:
Please catch the exception here and print something like "OK" in the `catch` block, and "Fail" otherwise.

Having a stacktrace in the expected output, like it is right now, is a bit inconvenient because the test data needs to be changed every time there's any slight refactoring to the affected files (`ClassReference.kt` in this case)

Reply:
With the new commit, the exception will be caught, and print a message : "KotlinReflectionNotSupportedError has been caught". Is this ok?